joomla/joomla-cms
# Improve captcha empty error message

Pull Request for Issue #18450.

### Summary of Changes
Captcha empty error message is vague/unclear. This PR improves the error message.


### Testing Instructions
Enable reCAPTCHA for Contacts form.
Fill out form.
Leave captcha blank.
Submit form.
See error message.

### Expected result
Empty CAPTCHA solution not allowed.


### Actual result
Empty solution not allowed.
